### DISCUSSION TITLE
2023년 04월 18일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#38] [General]
# 2023년 04월 18일

LOG 2단계 미션 PR을 보냈다. PR 보낼 때 질문을 많이 생각해보려다가 브라운이 리뷰어들이 질문에 대답하는 게 오래 걸린다고 해서 그냥 생각난 것 하나만 적었다. Bean을 만드는 기준에 대해서 질문했는데 아마도 답이 없는 질문이라 just 리…

https://cyma-s.github.io/2023-04-18/

**Comment by be-student** [comment]
역시 베로베로베로베로 답네요

**Reply by Cyma-s** [reply]
베?로?베?로?

**Comment by gitchannn** [comment]
하이베로
나도 기침나다가 눈물 쏠리는거 먼지 알아
나도오늘 공부하다가 프록시 나왔는데 이름보고 쫄아서 도망쳣어 내일 나도 알려줄 수 있겠니!

**Reply by Cyma-s** [reply]
늦게 봤군..ㅋㅋㅋㅋ 근데 프록시는 나도 잘 모르겠는데 좀 더 공부하고 알려줄게

**Comment by kevstevie** [comment]
세터주입 쓰지마세요

**Reply by Cyma-s** [reply]
세터 주입은 어떤 부분에서는 좋다 - spring.io 공식 문서

**Reply by kevstevie** [reply]
쓰지말라면 쓰지마세요

**Reply by Cyma-s** [reply]
더 쓰고 싶네요~?

**Comment by drunkenhw** [comment]
굿

**Reply by Cyma-s** [reply]
천재 개발자님이 들러주시다니 영광입니다~
